accounts/accounts.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2015 The go-ethereum Authors
```

Comment:
Well, this file started out in 2015, just that it was split, so git figured one file is moved while the other is new. I think it should remain 2015 though as that's when it started out.